test/integration/content-type-test.js

```diff
@@ -16,7 +16,7 @@ buster.testCase('Content Type', {
       'order': 'name',
       'sys.id[in]': 'cat,dog,human'
     }).then(function(contentTypes) {
-      assert.match(contentTypes, [
+      assert.match(contentTypes.items, [
```

Comment:
yes, as commented in the commit message, this would break BC. The old API is not tenable.